stage0/run.go

```diff
@@ -452,6 +457,12 @@ func Run(cfg RunConfig, dir string, dataDir string) {
 		log.Fatalf("error clearing FD_CLOEXEC on lock fd")
 	}
 
+	tpmEvent := fmt.Sprintf("rkt: Rootfs: %s Manifest: %s Stage 1 args: %s", cfg.CommonConfig.RootHash, cfg.CommonConfig.ManifestData, strings.Join(args, " "))
+	// If there's no TPM available or there's a failure for some other
+	// reason, ignore it and continue anyway. Long term we'll want policy
+	// that enforces TPM behaviour, but we don't have any infrastructure
+	// around that yet.
+	tpm.Extend(tpmEvent)
```

Comment:
sorry, more things:
a) to explicitly flag the error is being ignored, please `_ = tpm.Extend(tpmEvent)`
b) after digging into the actual details - I'm a bit scared of this being in the critical path when it involves an HTTP request to an endpoint that in most cases is going to be unavailable (or running who knows what). Any ideas to mitigate that?